apps/right-to-rent-check/views/step.html

```diff
@@ -0,0 +1,8 @@
+{{<partials-page}}
+  {{$page-content}}
+    {{#fields}}
+      {{#renderField}}{{/renderField}}
+    {{/fields}}
+    {{#input-submit}}continue{{/input-submit}}
+  {{/page-content}}
+{{/partials-page}}
```

Comment:
You were right to remove it as it doesn't need it.  New PR https://github.com/UKHomeOffice/right-to-rent-check/pull/3